### DISCUSSION TITLE
Midi input issues

### DISCUSSION THREAD

**Discussion by fieldfra** [GrandOrgue/grandorgue#1826] [Help]
Hello everyone,

When I connect my midi organ to GrandOrgue and set it up, it works fine but once a key is pressed, the sound doesn't stop until you press the 'panic' button or deselect the key. Do you know what is causing this and how to fix it? 

Thanks a lot.

**Comment by oleg68** [comment]
Right click on the manual, in the midi receiving tab change Min Velocity from 0 to 1
